### PR TITLE
[MFD-6389] Migrate Belvedere to Android 13

### PR DESCRIPTION
**Pull request by ReguloSar**
### Changes
This PR's changes allows supporting Android 13 version (API 33), which brings new runtime permissions that are required to access media files, where we have to request them individually ( Images, Video, Audio ) as known as granular permissions. 

# Allowing permission flow:

https://user-images.githubusercontent.com/85873565/181457089-a7373904-3011-4292-8289-3f870e6d2de9.mp4

# Denying music and audio permission flow:

https://user-images.githubusercontent.com/85873565/181457877-72d63b13-6208-4111-9a36-590bb7d19d1e.mp4

### References
- [Jira](https://zendesk.atlassian.net/browse/MFD-6389)

### Risks
- Low

### CODE REVIEW

**Review by loyannec — APPROVED** [review]
Nice great job, Antonio 🚀

File: belvedere/src/main/java/zendesk/belvedere/ImageStreamPresenter.java [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+import java.util.Arrays;
 import java.util.List;
-
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import zendesk.belvedere.PermissionManager.InternalPermissionCallback;
 import zendesk.belvedere.ui.R;
 
 class ImageStreamPresenter implements ImageStreamMvp.Presenter {
```
Comment (IanArb) [inline belvedere/src/main/java/zendesk/belvedere/ImageStreamPresenter.java:23]: Do we need to update the unit tests for these changes?

Comment (ReguloSar) [reply belvedere/src/main/java/zendesk/belvedere/ImageStreamPresenter.java:23]: I don't think we need to update but add an additional one. I'll double check this. Thanks Ian!

Comment (ReguloSar) [reply belvedere/src/main/java/zendesk/belvedere/ImageStreamPresenter.java:23]: Hey @IanArb . I doubled check this. I should be adding UI tests for these changes, but this repo doesn't have any UI tests so I'll be skipping this for now.

**Review by IanArb — APPROVED** [review]

**Pull request merged**